ArgoCD: Add default registry documentation (#2855)
OctopusDeploy/docs: src/pages/docs/argo-cd/steps/update-application-image-tags.md

more information.
:::

> :::div{.info}
> If your cluster's default registry has been overridden see [cluster annotations](/docs/argo-cd/annotations/cluster-annotations) to ensure

Reviewer: ```suggestion
If the application cluster's default registry has been changed see [cluster annotations](/docs/argo-cd/annotations/cluster-annotations) to ensure
```
